Code/Framework/AzCore/AzCore/Math/IntersectSegment.h

```diff
@@ -240,9 +240,24 @@ namespace AZ
         //! @return                    The number of intersecting points.
         int IntersectRayCone(
             const Vector3& rayOrigin, const Vector3& rayDir,
-            const Vector3& coneApex, const Vector3& coneDir, float coneHeight, float coneBaseRaidus,
+            const Vector3& coneApex, const Vector3& coneDir, float coneHeight, float coneBaseRadius,
             float& t1, float& t2);
 
+        //! Returns 1 if ray intersects the cone, 0 otherwise.
+        //! @note Does not give reliable results if the ray origin is inside the cone.
```

Comment:
We may want to give more information about why a user should choose version 1 vs version 2.

Reply:
I'm going to take the approach @amzn-mike suggested and just this move this closer to the usage code that needs it along with doc/comments to explain why it's needed, thanks for the suggestion though.